providers/facebook/facebook.go

```diff
@@ -60,6 +62,13 @@ func (p *Provider) SetName(name string) {
 	p.providerName = name
 }
 
+// SetCustomFields is to update the fields for endpointProfile
+// Fields values are referd with https://developers.facebook.com/docs/graph-api/reference/user
```

Comment:
This comment is a bit hard to understand. It should be updated to say something like,

```go
// SetCustomFields sets the fields used to return information
// for a user.
//
// A list of available field values can be found at
// https://developers.facebook.com/docs/graph-api/reference/user
```

Reply:
OK. I will change the comment.